ucm2/platforms/msm8916/qdsp6-components.conf

```diff
@@ -1,4 +1,13 @@
 If.qdsp6 {
+	Condition {
+		Type String
+		Haystack "${var:IsQuinary}"
+		Needle "yes"
+	}
```

Comment:
Perhaps make this a bit easier

```
Condition {
	Type String
	Empty "${var:MultiMedia3Port}"
}
True.Define.MultiMedia3Port "QUAT_MI2S_RX"
```

then in ocean HiFi

```
Define {
	...
	MultiMedia3Port "QUIN_MI2S_RX"
}
```

Would this work?

Reply:
Done.